radicle-seed/src/main.rs

```diff
@@ -0,0 +1,46 @@
+use std::net;
+
+use librad::peer::PeerId;
+use radicle_seed::{Mode, Node, NodeConfig};
+
+use argh::FromArgs;
+use futures::executor;
+use log;
+
+#[derive(FromArgs)]
+/// Radicle Seed.
+pub struct Options {
+    /// track the specified peers only
+    #[argh(option)]
+    pub track_peers: Vec<PeerId>,
+
+    /// listen on the following address for peer connections
+    #[argh(option)]
+    pub listen: Option<net::SocketAddr>,
+
+    /// log level (default: info)
+    #[argh(option, default = "log::Level::Info")]
+    pub log: log::Level,
+}
+
+impl Options {
+    pub fn from_env() -> Self {
+        argh::from_env()
+    }
+}
+
+fn main() {
```

Comment:
```suggestion
#[tokio::main]
async fn main() {
```

This will give you an executor and you can await inside your main. EMBRACE LE ASYNC

Reply:
This was needed to run the executor :grimacing: